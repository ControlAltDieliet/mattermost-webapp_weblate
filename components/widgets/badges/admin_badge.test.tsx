// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import AdminBadge from './admin_badge';

describe('components/widgets/badges/AdminBadge', () => {
    test('should match the snapshot', () => {
        const wrapper = shallow(<AdminBadge className={'test'}/>);
        expect(wrapper).toMatchSnapshot();
    });
});
