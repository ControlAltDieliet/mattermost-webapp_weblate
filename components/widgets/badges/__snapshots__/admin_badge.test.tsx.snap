// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/widgets/badges/AdminBadge should match the snapshot 1`] = `
<Memo(Badge)
  className="AdminBadge test"
  show={true}
>
  <MemoizedFormattedMessage
    defaultMessage="ADMIN"
    id="post_info.admim"
  />
</Memo(Badge)>
`;
